pkg/secrets/secrets_test.go

```diff
@@ -0,0 +1,37 @@
+package secrets
+
+import (
+	"testing"
+
+	oconfig "github.com/openshift/api/config/v1"
+)
+
+func Test_processTags(t *testing.T) {
+	type args struct {
+		platformType oconfig.PlatformType
+		userData     string
+	}
+	tests := []struct {
+		name string
+		args args
+		want string
```

Comment:
nit: expected, instead of want would follow the naming conventions for the majority of the repo's unit tests

Reply:
Fixed